packages/legend-application-marketplace/src/pages/VendorData/LegendMarketplaceVendorData.tsx

```diff
@@ -0,0 +1,202 @@
+  (props: { marketPlaceVendorDataState: LegendMarketPlaceVendorDataState }) => {
+    const { marketPlaceVendorDataState } = props;
+
+    return (
+      <div className="legend-marketplace-vendordata-main">
+        <div className="legend-marketplace-vendordata-main-sidebar col-sm-4">
+          <div className="legend-marketplace-vendordata-main-sidebar__title">
+            Filters
+          </div>
+          <hr></hr>
+          <div className="legend-marketplace-vendordata-main-sidebar__subtitle">
+            Providers
+          </div>
+          <div className="legend-marketplace-vendordata-main-content__sidebar__checkbox-filter-group">
+            {marketPlaceVendorDataState.dataFeedProviders.map((vendor) => (
+              <div
+                key={vendor.id}
+                className="legend-marketplace-vendordata-main-sidebar__checkbox-filter"
+              >
+                <input type="checkbox" value={vendor.providerName} />
+                <label>{vendor.providerName}</label>
+              </div>
+            ))}
+          </div>
+        </div>
+        <div className="legend-marketplace-vendordata-main-search-results">
+          <SearchResultsRenderer
+            providerResults={marketPlaceVendorDataState.dataFeedProviders}
+            sectionTitle="Data Feed"
+          />
+          <hr></hr>
+          <SearchResultsRenderer
+            providerResults={marketPlaceVendorDataState.terminalProviders}
+            sectionTitle="Terminal License"
+          />
+          <hr></hr>
+          <SearchResultsRenderer
+            providerResults={marketPlaceVendorDataState.addOnProviders}
+            sectionTitle="Add-Ons"
+          />
+        </div>
+      </div>
+    );
+  },
+);
+
+export const LegendMarketPlaceVendorData = observer(
+  (props: { marketPlaceVendorDataState: LegendMarketPlaceVendorDataState }) => {
+    const { marketPlaceVendorDataState } = props;
+    const applicationStore = useApplicationStore();
+
+    const onChange = (
+      provider: string | undefined,
+      query: string | undefined,
+    ) => {
+      // Handle search logic here
+    };
+
+    const searchOptions = {
+      placeholder: 'Search data...',
+      // Add other search options here
+    };
+
+    useEffect(() => {
+      flowResult(marketPlaceVendorDataState.populateProviders()).catch(
+        applicationStore.alertUnhandledError,
+      );
+    }, [marketPlaceVendorDataState, applicationStore]);
+
+    return (
+      <div className="app__page">
+        <div className="legend-marketplace-vendor-data">
+          <LegendMarketplaceHeader />
+          <div className="legend-marketplace-vendor-data__content">
+            <div className="legend-marketplace-banner">
+              <div className="legend-marketplace-banner__title">
+                Vendor Data
+              </div>
+              <div className="legend-marketplace-banner__subtitle">
+                <p>Discover the high quality data</p>
```

Comment:
Nit: I don't really like this wording. Let's just say
```suggestion
                <p>Discover high quality data</p>
```